src/Amalgam/GeneralizedDistance.h

```diff
@@ -1169,7 +1176,10 @@ class RepeatedGeneralizedDistanceEvaluator
 				double default_deviation = deviations_for_sid->second.defaultDeviation;
 				if(FastIsNaN(default_deviation))
 				{
-					deviations.defaultDeviation
+					feature_data.defaultNominalMatchDistanceTerm
```

Comment:
shouldn't default_deviation be set to something here though? otherwise it still remains a NaN